openshift/tests-extension/test/olmv1.go

```diff
@@ -30,8 +30,8 @@ var _ = Describe("[sig-olmv1] OLMv1", func() {
 })
 
 var _ = Describe("[sig-olmv1][OCPFeatureGate:NewOLM] OLMv1 CRDs", func() {
```

Comment:
please refer to https://github.com/openshift/operator-framework-operator-controller/pull/418#discussion_r2258671876